project.mk

```diff
@@ -0,0 +1,9 @@
+OPERATOR_NAME?=$(shell sed -n 's/.*OperatorName .*"\([^"]*\)".*/\1/p' config/config.go)
```

Comment:
Same thing here. Not an operator so do we need these?

Reply:
changes the variables